circle.yml

```diff
@@ -9,8 +9,6 @@ dependencies:
 test:
   pre:
     - tsc
-  post:
-    - mv coverage/* $CIRCLE_ARTIFACTS/ || exit_on_error "No coverage!"
```

Comment:
no codecov?

Reply:
first prio was having something working, given the size/exposure of the repo